packages/netlify-cms-backend-github/src/API.js

```diff
@@ -115,18 +119,12 @@ export default class API {
     return `${this.repo}/${collectionName}/${slug}`;
   }
 
-  generateBranchNameFromCollectionAndSlug(collectionName, slug) {
-    return this.generateContentKey(collectionName, slug).then(contentKey =>
-      this.generateBranchName(contentKey),
-    );
-  }
-
   generateBranchName(contentKey) {
     return `${CMS_BRANCH_PREFIX}${contentKey}`;
   }
 
   branchNameFromRef(ref) {
-    return ref.substring('refs/heads/'.length - 1);
+    return ref.substring('/refs/heads/'.length - 1);
```

Comment:
Looks like a bug introduced in https://github.com/netlify/netlify-cms/pull/2430.
Without the forward slash after creating an entry (and refreshing the page), nothing will appear under `workflow` (the branch and the pr are created, just not displayed)